packages/edit-widgets/src/index.js

```diff
@@ -3,6 +3,7 @@
  */
 import { render } from '@wordpress/element';
 import { registerCoreBlocks } from '@wordpress/block-library';
+import '@wordpress/format-library';
```

Comment:
Should we add the "package.json" and backend dependency?

Reply:
Hi @youknowriad, I changed the PR and applied the same approach used by edit-post. The format library is not imported and wp_enqueue* is used to load the scripts and styles. A package.json dependency was added as edit-post also contains it.